### DISCUSSION TITLE
requesting code to update package.

### DISCUSSION THREAD

**Discussion by NaillaUwase** [rwema3/RecipeApp#2] [Q&A] [answered]
Hello Rwema, can you assist me Code to Update " webview_flutter" package that you've used in this project?

**Comment by rwema3** [comment]
Run this command in terminal:

 $ flutter pub add webview_flutter

This will add a line like this to your package's pubspec.yaml (and run an implicit flutter pub get):

dependencies:
  webview_flutter: ^3.0.4

**Reply by NaillaUwase** [reply]
Thanks

**Comment by rwema3** [comment, marked as answer]
Now in your Dart code, you can use:
import 'package:webview_flutter/webview_flutter.dart';

**Reply by NaillaUwase** [reply]
thank you so much
